src/components/Characters/Character.css

```diff
@@ -3,16 +3,16 @@
 }
 
 .Character__image{
```

Comment:
let's use kebab-case for css ^

Reply:
sure, I believe now I have a "muscle memory" about using it